paramiko_expect.py

```diff
@@ -65,6 +65,12 @@ def __init__(
     def __del__(self):
         self.close()
 
+    def __enter__(self):
```

Comment:
nice touch !!

you think you can also mention this in the readme ? how to use as context ?